src/firstboot/firstboot.c

```diff
@@ -832,6 +839,7 @@ static int help(void) {
                "     --setup-machine-id        Generate a new random machine ID\n"
                "     --force                   Overwrite existing files\n"
                "     --delete-root-password    Delete root password\n"
+               "     --root-password-is-hashed Indicate that the root password has already been hashed\n"
```

Comment:
needs to be updated, too

(and should be moved to the two `--root-password=` lines above)